core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/workflow/PhysicalPlan.scala

```diff
@@ -60,6 +61,28 @@ case class PhysicalPlan(
 
   def getSinkOperators: List[LayerIdentity] = this.sinkOperators
 
+  def findLayerForInputPort(opName: OperatorIdentity, portName: String): LayerIdentity = {
+    val candidateLayers = layersOfLogicalOperator(opName).filter(op =>
+      op.inputPorts.map(_.displayName).contains(portName)
+    )
+    assert(
```

Comment:
This assertion is causing one of my workflows containing a Word Cloud (V1) operator to fail.